src/main/app/src/pages/BrowsePage.js

```diff
@@ -96,7 +99,9 @@ const BrowsePage = ({ links, activityType, updateActivityType, updateActivity, u
 
         {/* This division is for filtering the results based on the user entered input. */}
         <ActivitiesFilterBar activityType={activityType} activityTypes={activityTypes}
-          links={links} updateFilteredLinks={updateFilteredLinks} />
+          links={links} updateFilteredLinks={updateFilteredLinks} updateGameError={updateGameError}/>
+
+        {gameError ? (<h6 style={{color: 'red'}}>Error: invalid number, please enter a positive integer.</h6>) : <div/>}
```

Comment:
I think using a material UI alert here would be best. It's what is being used on other pages so it would be best for uniformity.

Reply:
Sure -- will change that now!